core/include/kernel/interrupt.h

```diff
@@ -233,7 +232,10 @@ static inline void interrupt_unmask(struct itr_chip *chip, size_t itr_num)
  */
 static inline void interrupt_enable(struct itr_chip *chip, size_t itr_num)
 {
-	chip->ops->enable(chip, itr_num);
+	if (chip->ops->enable)
+		chip->ops->enable(chip, itr_num);
+	else
+		interrupt_unmask(chip, itr_num);
```

Comment:
Wouldn't it make sense for the driver to supply the unmask function instead of the enable function if it doesn't have an enable function?

Reply:
Okay, i'll drop commit "core: interrupt: enable/disable fallback to mask/unmask".